### ISSUE TITLE
[Question] E2EE implications of using ntfy for push notifications

### ISSUE THREAD

**Issue by opusforlife2** [vector-im/element-android#6574]
The ntfy server does not have E2EE support yet (https://github.com/binwiederhier/ntfy/issues/69).

Element is E2EE though, so does that mean ntfy's lack of E2EE is irrelevant? Or is there some data that could be collected from the push notifications sent through ntfy?

I hope this information could be added to a wiki somewhere.

**Comment by karmanyaahm** [comment]
> Element is E2EE though, so does that mean ntfy's lack of E2EE is irrelevant

Yes, your messages cannot be read by anyone except the end devices. Additionally, Element only sends random message and room IDs over the push notifications. The message contents are fetched from the homeserver separately.

> is there some data that could be collected from the push notifications sent through ntfy

Primarily just when and how many notifications you receive, which can be eliminated by self-hosting a ntfy server.

**Comment by opusforlife2** [comment]
Thanks! That is very reassuring.
